.github/workflows/loristest.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       uses: shivammathur/setup-php@v2
```

Comment:
seems like php-ast extension might be getting installed twice ?

Reply:
yeah, looks like it. Remove the `sudo` part

Reply:
do you know the fix for git actions ? is cant find php8-ast

Reply:
on the main branch it's just `php-ast`. Maybe it's easiest to just copy `.github/workflows/loristest.yml` from the main branch? Any fixes to it on the release branch were just to fix the release branch and it's currently passing on main..